evm-development/quickstart-guide.md

```diff
@@ -29,6 +47,8 @@ _Explorer:_ [_https://evm.explorer.canto.io/_](https://evm.explorer.canto.io/)
 **Chain ID**: **7701**
 
 _Explorer:_ [_https://testnet.tuber.build/_](https://testnet.tuber.build/)
+**Alternative Testnet Explorer URL:** [https://canto-test.dex.guru/](https://canto-test.dex.guru/)
```

Comment:
Could you change this to match the formatting on the previous line? i.e. italics for the whole line